tools/buffer_points.py

```diff
@@ -1,10 +1,8 @@
 
-
 def buffer_addresses(point_gdf, buffer_distance):
-    # your code here
-
     # project to 3857
-    
+    point_gdf.to_crs(epsg=3857)
     # buffer by buffer distance
-
+    point_gdf['geometry'] = point_gdf.buffer(buffer_distance)
+    pts = point_gdf
```

Comment:
@taikingm here you can return point_gdf at end of fucntion, you can drop the `pts` from the return, 

So just have your return look like this:

```python

return points_gdf

```

Reply:
Done